### ISSUE TITLE
[ACCESSIBILITY] Make all functionality available from a keyboard

### ISSUE THREAD

**Issue by markusmann-vg** [SORMAS-Foundation/SORMAS-Project#9642]
### Feature Description
All [functionality](https://www.w3.org/TR/WCAG21/#dfn-functionality) of the content is operable through a [keyboard interface](https://www.w3.org/TR/WCAG21/#dfn-keyboard-interface) without requiring specific timings for individual keystrokes, except where the underlying function requires input that depends on the path of the user's movement and not just the endpoints.

**NOTE**
This exception relates to the underlying function, not the input technique. For example, if using handwriting to enter text, the input technique (handwriting) requires path-dependent input but the underlying function (text input) does not.

**NOTE**
This does not forbid and should not discourage providing mouse input or other input methods in addition to keyboard operation.

### Proposed Change
Table elements cannot be selected without the mouse. A selection and an opening of the edit mask is therefore not possible. A solution here could be to implement a focusable button in the table rows.
- Providing keyboard-triggered event handlers
- Ensuring keyboard control for all functionality

### Implementation Details
- Providing keyboard-triggered event handlers (https://www.w3.org/WAI/WCAG21/Techniques/general/G90.html)
- Ensuring keyboard control for all functionality (https://www.w3.org/WAI/WCAG21/Techniques/general/G202.html)

### Additional Information
[](https://www.w3.org/TR/WCAG21/#keyboard-accessible)
